docs/lists_degrees.md

```diff
@@ -143,7 +143,7 @@ This list is [autocomplete compatible](autocomplete_compatability.md).
 | `suggestion_synonyms` | string array | A list of common alternative names that *may* be appropriate for this subject |
 | `match_synonyms` | string array | A list of common alternative names that are equivalent to this subject. An answer matching a match synonym can be safely matched to this subject.|
 | `dttp_id` | uuid | The ID used for this subject in DTTP |
-| `hecos_code` | string | The ID used for this subject in [HECoS](https://www.hesa.ac.uk/support/documentation/hecos) |
+| `hecos_code` | string | The HESA [DEGSBJ](https://www.hesa.ac.uk/collection/c22053/e/degsbj) code for this degree subject |
```

Comment:
```suggestion
| `hecos_code` | string | The HESA [DEGSBJ](https://www.hesa.ac.uk/collection/c22053/e/degsbj) code for this degree subject |
```

Unsure about this one - it's a HECOS code, not a HESA code. We could possibly link to HECOS here instead or in addition?

Reply:
I'll do "in addition", I reckon!